go/interpreter/lfvm/opcode_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+import (
+	"fmt"
+	"slices"
+	"testing"
+)
+
+func TestOpcode_String(t *testing.T) {
+	for i := 0; i < 256; i++ {
+		op := OpCode(i)
+		want, found := op_to_string[op]
+		got := op.String()
```

Comment:
The `String` function uses the `op_to_string` function internally. This test is testing whether the same function returns the same result when called with the same argument.